src/scores.ts

```diff
@@ -23,6 +23,14 @@ async function getBlockNum(network) {
 
 async function calculateScores(args, key) {
   const { space = '', strategies, network, addresses } = args;
+
+  const key = createHash('sha256')
```

Comment:
@bonustrack @ChaituVR This last merge commit messed things up a bit. This here won't work. (`key` already declared)

Reply:
Fixed it